src/main/resources/api/service_openapi.yaml

```diff
@@ -1,264 +1,231 @@
-openapi: 3.0.0
+openapi: 3.0.3
 info:
   title: Workspace Manager API
-  description: |
-    Workspace Manager REST API to manage workspaces.
+  description: Workspace Manager REST API to manage workspaces.
   version: 0.0.1
 
-servers:
-- url: /
-
 paths:
-  '/status':
+  # Unauthorized Paths
+  /status:
     get:
-      description: |
-        Returns the operational status of the service
+      summary: Returns the operational status of the service
       operationId: serviceStatus
-      tags:
-      - unauthenticated
+      tags: [Unauthenticated]
       responses:
-        200:
+        '200':
           description: Service is functional
-          $ref: '#/components/responses/StatusResponse'
-        500:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SystemStatus'
+        '500':
           description: Service is broken
-          $ref: '#/components/responses/StatusResponse'
-  '/version':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SystemStatus'
+  /version:
     get:
-      description: Returns the deployed version of the service.
+      summary: Returns the deployed version of the service
       operationId: serviceVersion
-      tags:
-      - unauthenticated
+      tags: [Unauthenticated]
       responses:
-        200:
+        '200':
           $ref: '#/components/responses/VersionResponse'
 
-  '/api/workspaces/v1':
+  # Workspace paths
+  /api/workspaces/v1:
     post:
-      description: |
-        Create a new Workspace. This should only be called by Folder Manager.
+      summary: Create a new Workspace. This should only be called by Folder Manager.
       operationId: createWorkspace
-      tags:
-      - workspace
+      tags: [Workspace]
       requestBody:
-        description: |
-          Auth token, spend profile, and policy information from
-          Folder Manager, as well as async job information.
+        required: true
```

Comment:
openapi-generator doesn't support the `requestBodies` field for refs, for some reason, but if we switch to swagger codegen, the schemas for these can move to that.